README.md

```diff
@@ -93,6 +93,20 @@ For the full list of commands, please see the [CLI command reference](https://do
 
 1. Install Node.js [`node@12.x` and `npm@6.x` are required]
 1. Clone this repository `$ git clone git@github.com:particle-iot/particle-cli.git && cd ./particle-cli`
+1. Install dependencies required by some of particle-cli's npm dependencies
+
+   >  particle-cli uses npm packages [`node-usb`](https://github.com/tessel/node-usb) and [`node-serialport`](https://github.com/serialport/node-serialport).
+   >  On some environments (ie: Raspberry Pi and Apple M1), `node-serialport` [does not provide prebuild environments](https://github.com/serialport/node-serialport/issues/2292),
+   >  so their npm install script relies on creating a prebuild using node-gyp (See
+   >  [node-gyp requirements](https://github.com/nodejs/node-gyp#on-unix)), and libudev (See [node-usb installation](https://github.com/tessel/node-usb#installation).
+   >  And these must be installed in order for their npm install script to succeed.  As well, particle-cli uses dfu-util and openssl.
+   >
+   >  As an example, to install these dependencies on Raspbian/Debian/Ubuntu:
+   >  ```bash
+   >  sudo apt update && sudo apt upgrade
+   >  sudo apt install build-essential libudev-dev python3 dfu-util openssl
+   >  ```
```

Comment:
It's probably useful in both places.  For local development and end-user installation, users will get stuck if on ARM device. I scratched my head for awhile until I traced it to these missing dependencies.

Reply:
I also think particle should iterate https://raw.githubusercontent.com/spark/particle-cli-wrapper/master/installer/unix/install-cli.  Checks for the node version and these dependencies may be best inside the install script rather than the instructions.